scripts/install

```diff
@@ -21,13 +20,45 @@ elif [[ $RPI_REVISION == *"00"* ]]; then
     echo ">>> Starting installation on Raspberry Pi 1..."
 elif [[ $RPI_REVISION == *"a01041"* || $RPI_REVISION == *"a21041"* ]]; then
     echo ">>> Starting installation on Raspberry Pi 2..."
-elif [[ $RPI_REVISION == *"a02082"* || $RPI_REVISION = *"a22082"* ]]; then
+elif [[ $RPI_REVISION == *"a02082"* || $RPI_REVISION == *"a22082"* ]]; then
     echo ">>> Starting installation on Raspberry Pi 3..."
 else
     echo ">>> Unknown Raspberry Pi model"
     exit 1
 fi
 
+# Prompt and set missing flags
+if [ -z "$WITH_MESH_POINT" -o "$WITH_MESH_POINT" != "true" -a "$WITH_MESH_POINT" != "false" ]; then
+    read -p "Configure Mesh Point interface (Y/n)? " CHOICE_MESH_POINT;
+    if [ "$CHOICE_MESH_POINT" == "Y" ]; then
```

Comment:
- Should accept `y`, `Y`, `Yes`, `yes`
- The prompt has a capital `Y` suggesting that is the default. An empty response returns the `else` statement, which I think is incorrect.